cookbooks/aws-parallelcluster-config/recipes/nfs.rb

```diff
@@ -16,11 +16,7 @@
 # limitations under the License.
 
 # For performance, set NFS threads to max(num_cores, 8)
```

Comment:
align comment with the new value logic

Reply:
Comment removed, it is in the default.rb now. Thanks!